packages/validator/package.json

```diff
@@ -26,6 +28,7 @@
   },
   "devDependencies": {
```

Comment:
You're missing `rimraf` as a dev dependency.

Reply:
Added in ff66de5eef05f1fb88e6ea103cb1817d27844f3d